test/portworx/testValues/storagecluster_all_components_enabled.yaml

```diff
@@ -8,7 +8,10 @@ storkSpec:
     mountPath: /etc/kubernetes
     secret:
       configMap: stork-config
-csi: true
+csi: 
+  enabled: true
+  topology: true
```

Comment:
same comment as above, topology is an object here which has enabled boolean field. I see below it is added correctly.

Reply:
We added `topology: true` in `values.yaml` for simplicity because there are no nested configurations for topology. In the `storageCluster` template, it gets converted to `topology.enabled: true` to ensure it works as expected. This approach keeps it straightforward for users to enable topology without dealing with additional nesting.

Reply:
Done, @nikita-bhatia  . I have added `topology.enabled` as per our discussion.

Reply:
Thanks! As discussed, it would be easier to manage these fields going forward in case any new fields are added to Topology struct in operator. So we will not have to change the type back to object in helm upgrade then.